src/components/DpEditor/MentionList.vue

```diff
@@ -88,6 +91,9 @@ export default {
         this.command(item)
       }
     }
+  },
+  mounted () {
+    console.log('MentionList mounted')
```

Comment:
non-blocking: please remove console.log

Reply:
thanks for spotting
9aaeca732505f95bcc3eaeac57f3c62f4a2036c2